libminifi/include/agent/CPPLINT.cfg

```diff
@@ -1,3 +1,3 @@
 set noparent
```

Comment:
Not sure why we have noparent setting here. The content just seems to duplicate the root cppling.cfg, so I think both line 1 and 2 can be removed from this file.

Reply:
It was indeed a duplicate, modified as requested.